data/json/npcs/BGSS_talk_professions.json

```diff
@@ -0,0 +1,409 @@
+      { "text": "Let's talk about something else.", "topic": "TALK_DONE" }
+    ]
+ },
+     {
+    "id": "BGSS_HOSPITAL_2_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "A young mother. I know she was a mother, because I delivered her baby. Sweet girl, she... she had a good sense of humour. She came in, spitting that black goo, fighting the orderlies, dead from a bullet wound through the chest. That's when I ... I don't know if I woke up, finally, or if I finally went crazy. Either way, I broke. I broke a lot earlier than my colleagues, and that's the only reason I lived. I skipped out, went to a dead corner of the hospital I used to hide in when I was a resident. An old stairwell leading to a closed-off unit the maintenance staff were using to store old equipment. I hid there for hours, while I listened to the world crumbling outside and inside.",
+    "responses": [
+      { "text": "How did you get out of there?", "topic": "BGSS_HOSPITAL_2_STORY3" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+      {
+    "id": "BGSS_HOSPITAL_2_STORY3",
+    "type": "talk_topic",
+    "dynamic_line": "Somehow, I don't know how, I managed to fall asleep in there. I think it might have started with me hyperventilating and passing out. When I woke up it was night, I was starving and parched, and... and the screaming had died down. At first I tried to go out the way I came in, but I peaked out the window and saw one of the nurses stumbling around, spitting that black shit up. Her name was Becky. She wasn't Becky anymore. So, I went back up and somehow made it into the storage area. From there, the roof. I drank water from some nasty old puddle and I camped out there for a while, watching the city around me burn	",
+    "responses": [
+      { "text": "How did you get out of there?", "topic": "BGSS_HOSPITAL_2_STORY3" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+
+      {
+    "id": "BGSS_HOSPITAL_2_STORY4",
+    "type": "talk_topic",
+    "dynamic_line": "Well, I still didn't have any food. Eventually I had to climb down the side of the building... so I did, as quietly as I could. It was night, and I have pretty good nightvision. Apparently the zombies don't, because I was able to slip right past them and steal a bicycle that was just laying on the side of the road. I'd kind of scouted out my route from above... I'm not from a big city, the hospital was the tallest building around. I avoided the major military blockades, and headed out of town towards a friend's old cabin. I had to fight off a couple of the zombies, but I managed to avoid any big fuss, by some miracle. I never made it to the cabin, but that's not important now.",
+    "responses": [
+      { "text": "What did you see, up there on the roof?", "topic": "BGSS_HOSPITAL_2_ROOF1" },
+    "//": "In the future this NPC should give you the map coordinates to the cabin if you persuade.",
```

Comment:
I see it. Already fixed in my home document, along with the other one a little ways down.